ADOPTERS.md

```diff
@@ -195,6 +195,22 @@ Details (optional):
 - 25000 samples/s
 - 1.2M active series
 
+## SUSE Rancher
+
+[suse.com/products/suse-rancher](https://www.suse.com/products/suse-rancher/)
+
+Environments: RKE, RKE2/k3s, AWS, Azure, Google Cloud, Bare Metal, etc.
+
+Uses [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus): Yes
+
+Rancher currently maintains patches on the [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) Helm chart to support use cases for Prometheus Operator across various different cluster types and setups (including Windows clusters) that are managed via the Rancher product. All Rancher users that install Monitoring V2 deploy this chart. 
+
+For more information, please see https://rancher.com/docs/rancher/v2.6/en/monitoring-alerting/how-monitoring-works/.
+
+Rancher's open-source patched Helm chart, `rancher-monitoring`, is maintained in the `release-v2.*` branches of [rancher/charts](https://github.com/rancher/charts).
```

Comment:
I'd say both of these paragraphs are a bit too much info. We could just add them as links.

Reply:
simplified